src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const appropriateMan = people.filter((person) => {
```

Comment:
'Man' - singular form, 'men' - plural form
```suggestion
  const appropriateMen = people.filter((person) => {
```